tools/osbuilder/rootfs-builder/ubuntu/config.sh

```diff
@@ -32,4 +32,5 @@ INIT_PROCESS=systemd
 ARCH_EXCLUDE_LIST=()
 
 [ "$SECCOMP" = "yes" ] && PACKAGES+=" libseccomp2" || true
-[ -n "$SKOPEO_UMOCI" ] && PACKAGES+=" ca-certificates libgpgme11" || true
+[ "$SKOPEO" = "yes" ] && PACKAGES+=" libgpgme11" || true
+PACKAGES+=" ca-certificates"
```

Comment:
Technically `ca-certificates` is still only required for CC, not general Kata

Reply:
Yeah, it's one of those tricky ones as we are are delivering to the CCv0 branch, but you are correct if we are considering merging this to `main`. I'm can add a `CC` parameter to `rootfs.sh` if you think that is the best approach?

Reply:
If I do add a CC parameter do you think it is far to replace the `UMOCI` one with it as that is currently mandatory for CC, but maybe I'm painting myself into a bit of a corner doing that for the sake of just saving another option to set.

Reply:
Hmm yeah I had bound this to `SKOPEO_UMOCI` but even binding it to `UMOCI` isn't super clear or future-proof, but another variable is not very nice either. My suggestion would be to add it to the `PACKAGES` in l. 15 instead for now.

Reply:
Also I could take the same approach with AA_KBC and force CC on if skope, umoci, or AA_KBC are set.

Reply:
Cool. I've taken this approach and added a comment, so at least we are aware if/when we merge this code to `main`. Thanks for the suggestion :)